src/components/timeline/OverflowEdgeIndicator.js

```diff
@@ -13,6 +13,7 @@ type Props = {
   className: string,
   children: React.Node,
   panelLayoutGeneration: number,
+  intialSelected: HTMLElement,
```

Comment:
I'm not sure how this is type checking correctly, but I think it should be `initialiSelected: HTMLElement | null`.